Formula/watchman.rb

```diff
@@ -23,7 +29,8 @@ def install
                           "--with-pcre",
                           # we'll do the homebrew specific python
                           # installation below
-                          "--without-python"
+                          "--without-python",
+                          "--enable-statedir=#{var}/run/watchman"
```

Comment:
Actually, it's created by the formula at build time. So not sure what to do about that. If `/usr/local/var` exists at the beginning and it creates `/usr/local/var/run` and `/usr/local/var/run/watchman` during the build, do we then need to `rm -rf /usr/local/var/run` at the end of the build in `install`, and then `mkdir_p /usr/local/var/run/watchman` during `post_install`? But if at the beginning `/usr/local/var/run` already existed, and during `install` the build only creates `/usr/local/var/run/watchman`, then do we need to `rm -rf  /usr/local/var/run/watchman` only and then during post_install `mkdir /usr/local/var/run/watchman`?